modules/rosa-sts-creating-cluster.adoc

```diff
@@ -191,7 +191,7 @@ $ ccoctl aws create-iam-roles \
 $ for role in `find . -name "*-role.json"`
 do
   policy=$(sed -e 's/05-/06-/' -e 's/role/policy/' <<< ${role})
-  role_name=$(grep --color=never -o "RoleName\":\"(\w|-)*" ${policy} | sed "s/RoleName\":\"//")
+  role_name=$(grep --color=never -o RoleName.* ${policy} | sed "s/RoleName\":\"//" | sed "s/\"}//")
```

Comment:
I see this change should be used for for the latest ccoctl tool. But for the old versions those had been built, the regex cannot work, should we add some explanation here?